pkg/noderesourcetopology/filter.go

```diff
@@ -79,13 +89,17 @@ func resourcesAvailableInAnyNUMANodes(numaNodes NUMANodeList, resources v1.Resou
 			}
 
 			resourceBitmask.Add(numaNode.NUMAID)
+			klog.V(6).InfoS("feasible", "logKey", logKey, "node", nodeName, "NUMA", numaNode.NUMAID, "resource", resource)
 		}
 		bitmask.And(resourceBitmask)
 		if bitmask.IsEmpty() {
+			klog.V(4).InfoS("early verdict", "logKey", logKey, "node", nodeName, "resource", resource, "suitable", "false")
```

Comment:
make it V(6)?

Reply:
Here I'd like to have progressively more information, hence I picked V(4).
I now realized I'm not sure this matches how the verbosity levels in `klog` are intended to work, I'll doublecheck what the docs suggest and what other code in k8s does

Reply:
The angle I suggested V(6) (or V(5)) is based on:

1. Most prod cluster set the verbose level to 3 or 4.
2. I totally understand you want to log this information; however, Filter would run against basically all nodes, so it will generate a high volume of logs in a large cluster. With that being said, maybe either make them both at verbose level 5; or log at level 4 on only one case whichever yields less volume of log entries.

Reply:
The reason I'm inclined to keep this V(4) is that we expect to see this line, and the one right below, at most once per filter invovation, so it seems to me a good compromise. Would this address this concern or would it still be too spammy?

Reply:
> The reason I'm inclined to keep this V(4) is that we expect to see this line, and the one right below, at most once per filter invocation

Yes, per filter invocation, but against basically all nodes, which looks a bit spammy. How about V(5)?

Reply:
I'm fine with `V(5)`. Thanks!